### ISSUE TITLE
Device reboots with frida-ps - iOS 13.2.2 - Checkra1n JB

### ISSUE THREAD

**Issue by hduarte** [frida/frida#1079]
Device reboots upon frida-ps -Uai

**Device:**
- iPhone SE
- Jailbroken iOS 13.2.2 with Checkra1n jailbreak.

**Versioning (iOS device and macOS):**
```
frida --version
12.7.20
```
**Output:**
```
frida-ps -Uai
Failed to enumerate applications: the connection is closed
```

**Comment by oneohtrix** [comment]
Can confirm, here's the kernel panic log from serial:

```
panic(cpu 0 caller 0xfffffff01c7001a8): initproc exited -- no exit reason available -- (signal 6, exit status 0 )
Debugger message: panic
Memory ID: 0x6
OS version: 17B102
Kernel version: Darwin Kernel Version 19.0.0: Wed Oct  9 22:41:51 PDT 2019; root:xnu-6153.42.1~1/RELEASE_ARM64_S8000
KernelCache UUID: E5568684C25D539C58C559CD9B2AA4B6
Kernel UUID: C0C04BDB-6A5F-3FE0-AD0E-29F05BB687DB
iBoot version: iBoot-5540.40.51
secure boot?: YES
Paniclog version: 13
Kernel slide:     0x0000000015060000
Kernel text base: 0xfffffff01c064000
mach_absolute_time: 0x51b28fef9
Epoch Time:        sec       usec
  Boot    : 0x5dc9869b 0x00018d90
  Sleep   : 0x00000000 0x00000000
  Wake    : 0x00000000 0x00000000
  Calendar: 0x5dc98a16 0x00042c00

Panicked task 0xffffffe000384800: 885 pages, 6 threads: pid 1: launchd
Panicked thread: 0xffffffe004470fd8, backtrace: 0xffffffe03c8b3510, tid: 11782
                  lr: 0xfffffff01c154c90  fp: 0xffffffe03c8b3550
                  lr: 0xfffffff01c154af0  fp: 0xffffffe03c8b35c0
                  lr: 0xfffffff01c252e28  fp: 0xffffffe03c8b3660
                  lr: 0xfffffff01c11d5c4  fp: 0xffffffe03c8b3670
                  lr: 0xfffffff01c154468  fp: 0xffffffe03c8b39e0
                  lr: 0xfffffff01c1547b0  fp: 0xffffffe03c8b3a30
                  lr: 0xfffffff01c6ffe60  fp: 0xffffffe03c8b3a50
                  lr: 0xfffffff01c7001a8  fp: 0xffffffe03c8b3ab0
                  lr: 0xfffffff01c51bdc4  fp: 0xffffffe03c8b3b30
                  lr: 0xfffffff01c51b868  fp: 0xffffffe03c8b3ba0
                  lr: 0xfffffff01c536218  fp: 0xffffffe03c8b3c00
                  lr: 0xfffffff01c536520  fp: 0xffffffe03c8b3c50
                  lr: 0xfffffff01c14c6bc  fp: 0xffffffe03c8b3c80
                  lr: 0xfffffff01c11d978  fp: 0xffffffe03c8b3c90
                  lr: 0x00000001b9b9dec4  fp: 0x0000000000000000


** Stackshot Succeeded ** Bytes Traced 358208 **
wdog panic (attempt 1)
```

**Comment by MuchiMuchiPink** [comment]
My iPhone 7 kernel panics as well. I guess both of you are on checkra1n beta 0.9.**1** as well?
Because on the previous build it failed to enumerate too, but didn't panic.

**Comment by oleavr** [comment]
Fixed in Frida 12.7.21, just released.
